src/pages/functional-testing-framework/metadata.md

```diff
@@ -264,6 +268,154 @@ _Catalog/Data/CategoryData.xml_:
 
 As a result, the Functional Testing Framework sends an unauthorized POST request with an empty body to the `https://example.com/rest/V1/guest-carts` and stores the single string response that the endpoint returns.
 
+#### Company Relation Example
```

Comment:
#### `Creating a relation between two companies Example` would be clear

Reply:
this is the main heading before the create and delete heading